signer/local_cosigner.go

```diff
@@ -69,92 +31,162 @@ type LocalCosignerConfig struct {
 	RaftAddress string
 	Total       uint8
 	Threshold   uint8
+	//Localsigner LocalSoftSignThresholdEd25519Signature
 }
 
-type PeerMetadata struct {
-	Share                    []byte
-	EphemeralSecretPublicKey []byte
-}
+type LastSignStateStruct struct {
```

Comment:
nice! good way to encapsulate the sign state and it's associated mutex

Reply:
Thanks 👍